mesonbuild/modules/gnome.py

```diff
@@ -869,12 +869,13 @@ def _unpack_args(arg, kwarg_name, kwargs, expend_file_state=None):
         return []
 
     @permittedKwargs({'interface_prefix', 'namespace', 'object_manager', 'build_by_default',
-                      'annotations', 'docbook', 'install_header', 'install_dir'})
+                      'annotations', 'docbook', 'install_header', 'install_dir', 'sources'})
+                      'annotations', 'docbook', 'install', 'install_header'})
```

Comment:
__[Flake8]__

[E111] indentation is not a multiple of four


<!-- Comment by [SideCI](https://sideci.com) -->